handbook/people-ops/onboarding/index.md

```diff
@@ -22,20 +18,46 @@ At Sourcegraph, we don't think of teammates as resources — we think of them as
+  - Teammates based outside of U.S.
+     - **[Veem](https://veem.com/)**: Submit monthly invoices.
 - [Configure Google Calendar](https://calendar.google.com/calendar/r/settings)
   - Add your working hours to Google Calendar so your teammates know when it is safe to add meetings to your calendar.
   - Add any local [holidays](../holidays.md) you observe to your calendar so people know you're not working.
   - Make it so any attendees can edit meetings you create: `Default guest permissions: Modify event` (this makes it easy for teammates to reschedule when necessary).
-- Set up a recurring weekly 25-minute [1-1 meeting](../../leadership/1-1.md) with your manager (e.g., `1-1 $YOURNAME-$MANAGERNAME`).
-- If you are not familiar with Git, complete the [Git introduction](git_intro.md).
-- [Create an account on GitHub.com](https://github.com/join) if you don't already have one.
-- Add yourself to the [team page](../../../company/team/index.md).
-- Add a link to your team page bio to your Slack profile
-- Read the first page of [Sourcegraph's user documentation](https://docs.sourcegraph.com/user).
-- It might not be immediately necessary, but if you ever need to click a HubSpot link just ask for access in #sales.
-- Go through the [analytics onboarding](https://sourcegraph.looker.com/projects/sourcegraph_events/files/1_home.md). This onboarding is within Looker, so if you don't have an account you can ask for access in #analytics.
-- Read Sourcegraph's [information security policy](https://about.sourcegraph.com/security) and acknowledge your acceptance: https://forms.gle/LUK1YtwAMJLhtRPi6.
-- Watch Dan's [Sourcegraph Demo](https://drive.google.com/file/d/1VUZ0rnZQpNgjtGDI0tMC-h-OtL0Czz8H/view)
+  - Set up a recurring weekly 25-minute [1-1 meeting](../../leadership/1-1.md) with your manager (e.g., `1-1 $YOURNAME-$MANAGERNAME`).
+- Make sure you know how to contribute to the handbook
+  - [Create an account on GitHub.com](https://github.com/join) if you don't already have one.
+     - Share your username with your manager so they can add you to the Sourcegraph GitHub account and appropriate teams.
+  - If you are not familiar with Git, complete the [Git introduction](git_intro.md).
+  - Add yourself to the [team page](../../../company/team/index.md).
+     - Add a link to your team page bio to your Slack profile
+  - Read Sourcegraph's [information security policy](https://about.sourcegraph.com/security) and acknowledge your acceptance: https://forms.gle/LUK1YtwAMJLhtRPi6.
+- Learn about Sourcegraph
+  - 
+  - Read the first page of [Sourcegraph's user documentation](https://docs.sourcegraph.com/user).
+  - Go through the [analytics onboarding](https://sourcegraph.looker.com/projects/sourcegraph_events/files/1_home.md). This onboarding is within Looker, so if you don't have an account you can ask for access in #analytics.
+  - Watch Dan's [Sourcegraph Demo](https://drive.google.com/file/d/1VUZ0rnZQpNgjtGDI0tMC-h-OtL0Czz8H/view)
```

Comment:
Should values and strategy links be here instead of under the first step?

```suggestion
  - Read the first page of [Sourcegraph's user documentation](https://docs.sourcegraph.com/user).
  - Watch Dan's [Sourcegraph Demo](https://drive.google.com/file/d/1VUZ0rnZQpNgjtGDI0tMC-h-OtL0Czz8H/view)
  - Go through the [analytics onboarding](https://sourcegraph.looker.com/projects/sourcegraph_events/files/1_home.md). This onboarding is within Looker, so if you don't have an account you can ask for access in #analytics.
```

Reply:
Yep moved. I was on the fence, and the empty bullet is an artifact of that. Thanks for the suggestion, I've moved strategy and values here.